am-ui/webcomponents/create-ad-webcomponent/src/components/SwiperPreview.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <swiper
+    :loop="true"
+    navigation="true"
+    slides-per-view="2"
+    space-between="20"
+  >
+    <swiper-slide v-for="image of props.images">
+      <img :src="createImageFromTopicImage(image)" :alt="image.topic_image_id" />
+    </swiper-slide>
+  </swiper>
+</template>
+<script setup>
+import { Swiper, SwiperSlide } from 'swiper/vue';
+import 'swiper/css';
+
+const props = defineProps({ images: Array });
+
+/**
+ * @description
```

Comment:
Could you pleas add description for the createImageFromTopicImage function.

Reply:
I will do it, Thanks.